src/index.html

```diff
@@ -2,11 +2,371 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="./images/logo.png" alt="logotype">
+      </a>
+      <nav class="nav">
+        <li class="nav__item">
+          <a href="#apple" class="nav__link is-active">Apple</a>
+        </li>
+
+        <li class="nav__item">
+          <a href="#samsung" class="nav__link">Samsung</a>
+        </li>
+
+        <li class="nav__item">
+          <a href="#smartphones" class="nav__link">Smartphones</a>
+        </li>
+
+        <li class="nav__item">
+          <a href="#laptops&computers" class="nav__link"
+          data-qa="nav-hover">Laptops & computers</a>
+        </li>
+
+        <li class="nav__item">
+          <a href="#gadgets" class="nav__link">Gadgets</a>
+        </li>
+
+        <li class="nav__item">
+          <a href="#tablets" class="nav__link">Tablets</a>
+        </li>
+
+        <li class="nav__item">
+          <a href="#photo" class="nav__link">Photo</a>
+        </li>
+
+        <li class="nav__item">
+          <a href="#video" class="nav__link">Video</a>
+        </li>
+      </nav>
+    </header>
+    <main>
+      <div class="container">
+        <div class="card"
+        data-qa="card"
+        >
+          <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27"
+         >
+
+          <h2 class="card__list">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">Product code: 195434</p>
+
+
+          <div class="card__rate">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__reviews">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-price">Price:</p>
+            <p class="card__price-number">$2,199</p>
+          </div>
+
+          <div class="card__buy">
+            <a
+          href="#buy"
+          class="card__buy-now"
+          data-qa="card-hover"
+        >
+          BUY
+            </a>
```

Comment:
```suggestion
            <a
              href="#buy"
              class="card__buy-now"
              data-qa="card-hover"
            >
              BUY
            </a>
```
fix everywhere (for all tags)